packages/textfield/README.md

```diff
@@ -244,6 +244,9 @@ Additionally, it implements more features such as:
 
 By default, `<mwc-textfield>` will report validation on `blur`.
 
+**NOTE:** Every property or function that checks validity must await
```

Comment:
Maybe move this note directly into the description for `checkValidity` in the method table above, since it might be easy to miss it otherwise. Also clarify if you need to do that *every* time you call `checkValidity` or just the first time.